pkg/agent/agent.go

```diff
@@ -63,9 +64,13 @@ func Execute(porterCommand []string, porterHome string, porterConfig string) (er
 	}
 
 	// Remind everyone the version of Porter we are using
+	fmt.Fprintf(stderr, "porter version\n")
 	cmd := exec.Command(porter, "version")
-	cmd.Stdout = stdout
+	cmd.Stdout = stderr // send all non-bundle output to stderr
```

Comment:
Just curious, why do we want to send all non-bundle output to stderr?

Reply:
In general, I've learned that it helps to decide what a user should expect to see in stdout vs stderr up-front. For example, outputting a mix of debug into and the actual command output (like a json document) to stdout makes it impossible to parse. We are still correcting that in porter in some places (as part of our instrumentation efforts).

Thinking about what a user of the operator would expect to see in stdout for the porter agent container, I am assuming that they want the _output of the requested porter command_. So even though the agent is doing a few extra things, like here we are printing porter version for debugging, it makes it easier for someone to separate out debug info vs the actual command output if we send `porter version` to stderr.

Let me know if you see a problem with sending the output of version to stderr for this container!